helpdesk-bot/src/main/resources/application.yaml

```diff
@@ -48,3 +48,13 @@ app-authentication:
 http-client:
   connect-timeout: 45000
   read-timeout: 45000
+
+provisioning:
+  execute: false
+  user:
+    name: <user_with_provisioning_role>
```

Comment:
You should define here only default values. Settings that depends on the environment shouldn't be part of the application classpath.

Reply:
Done.